templates/zerver/help/mute-a-topic.md

```diff
@@ -1,43 +1,74 @@
 
-5. Upon clicking **Mute the topic (topic name)**, a tooltip titled **Topic Muted** will
-appear, and the topic will be grayed out in the stream list dropdown, confirming
-the success of your muting of the topic.
+3. Upon clicking **Mute the topic (topic name)**, a tooltip titled **Topic Muted**
+will appear, and the topic will be grayed out in the stream list dropdown,
+confirming the success of your muting of the topic.
 
-## Muting a topic through your view
+## Mute a topic through your current filter
 
-1. The default **Home** view displays all messages from unmuted streams and topics
-that you've subscribed to. Search for messages belonging to the topic that you
-wish to mute in the **Home** view.
+1. The default **Home** filter displays all messages from unmuted streams and topics
+that you are subscribed to. Find a message belonging to the topic that you wish
+to mute in your current filter or on the **Home** filter.
 
     !!! tip ""
-        Alternatively, you can narrow your view to messages sent to the topic
-        that you want to mute by entering the search operator `topic:(topic name)`
+        Alternatively, you can directly narrow your view to messages sent to the
+        topic that you want to mute by using the search operator `topic:(topic name)`
         in the search bar.
 
 {!message-actions.md!}
 {!down-chevron.md!}
 
-2. Select the **Mute the topic (topic name)** option from the dropdown
-   to mute the topic that the selected message belongs to.
+3. Select the **Mute the topic (topic name)** option from the dropdown to mute
+the topic that the selected message belongs to.
 
-4. Upon clicking **Mute the topic (topic name)**, a tooltip titled **Topic Muted** will
- appear, and any messages belonging to the topic will disappear from your view,
- confirming the success of your muting of the topic.
+4. Upon clicking **Mute the topic (topic name)**, a dialog titled **Topic Muted**
+will appear, and any messages belonging to the topic will disappear from your
+filter, confirming the success of your muting of the topic.
+
+## Unmute a topic through the stream list
+
+1. Find the **Streams** section located on the left sidebar. In the list, click
+on the stream that the topic you want to unmute belongs to. Upon clicking on the
+stream's name, all the topics of the stream will appear.
+
+2. In the stream's topics list, hover over the muted (and grayed out) topic to
+reveal a {!down-chevron.md!} To unmute the topic, simply select the
+**Unmute the topic (topic name)** option from the dropdown.
+
+3. Upon clicking **Unmute the topic (topic name)**, the topic will no longer be
+grayed out in the stream list dropdown, confirming the success of your muting
+of the topic.
+
+## Unmute a topic by narrowing to it
+
+1. Narrow to messages sent to the topic that you want to unmute by pressing the
+topic's name in the **Streams** section located on the left sidebar, or by
+using the search operator `topic:(topic name)` in the search bar.
```

Comment:
Well, I guess one would be smart enough to pick the right one. It is a bit like the tip above, it doesn't mention the possibility of it happening...